pkg/collect/http.go

```diff
@@ -127,10 +129,17 @@ func responseToOutput(response *http.Response, err error) ([]byte, error) {
 			headers[k] = strings.Join(v, ",")
 		}
 
+		var rawJSON json.RawMessage
+		if err := json.Unmarshal(body, &rawJSON); err != nil {
+			klog.Infof("failed to unmarshal response body as JSON: %v", err)
```

Comment:
should use klog.Errorf?

Reply:
I use `klog.Infof` because the response may not be JSON and in those cases we just log the message.

Reply:
I see, this log just needs to be logged.  It is not necessary to highlight it as error, since  the response may not be JSON